Timestepper revision: timesteppers as part of the models and allow for multiple ones (#125)
TUM-PIK-ESM/Terrarium.jl: docs/src/extending/state_variables.md

```
Variables are typically constructed using one of the convenience functions [`prognostic`](@ref), [`auxiliary`](@ref), or [`input`](@ref).

> These variable definitions are purely symbolic; they do not hold any data and cannot be used for computation. Calling [`StateVariables`](@ref) on a model, process, or [`Variables`](@ref) container (see following sections) results in corresponding [Fields](@ref) being allocated for each variable. 

Reviewer: ```suggestion
These variable definitions are purely symbolic; they do not hold any data and cannot be used for computation. Constructing [`StateVariables`](@ref) from a model, process, or [`Variables`](@ref) container (see following sections) results in corresponding [Fields](@ref) being allocated for each variable. 
```
